HaxeManual/12-target-details.tex

```diff
@@ -474,13 +429,13 @@ \subsection{Exposing Haxe classes for JavaScript}
 
 In this pattern, a var statement is used to expose the module; it doesn't write to the \ic{window} or \ic{exports} object. 
 
-\subsection{Loading extern classes using "require" function}
+\subsection{Loading extern classes using ``require'' function}
```

Comment:
Did you check if that generates to markdown correctly?

Reply:
Yup, it's fine.